frontend/packages/kubevirt-plugin/src/components/vm.tsx

```diff
@@ -43,30 +44,39 @@ const VMHeader: React.FC = (props) => (
     <ColHead {...props} className={otherRowSize} sortField="metadata.namespace">
       Namespace
     </ColHead>
-    {/* TODO(mlibra): migrate VM status in a follow-up
-  <ColHead {...props} className={mainRowSize} sortField="spec.running">State</ColHead>
-  */}
+    <ColHead {...props} className={mainRowSize} sortField="spec.running">
```

Comment:
We should update to PF4 tables. See #1465 and follow on #1707.

Reply:
ok, this is wip in #1758